tests/installation/hostname_inst.pm

```diff
@@ -0,0 +1,37 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use strict;
+use warnings;
+use base "y2logsstep";
+use testapi;
+
+sub run() {
+    my $self = shift;
+    assert_screen "before-package-selection";
+
+    #send_key "ctrl-alt-shift-x"; sleep 3;
+    select_console('install-shell');
+    
+    if (get_var('HOSTNAME')) {
+        assert_script_run 'test $(hostname) == "myhost"'
```

Comment:
so far the variable HOSTNAME is used as a boolean switch variable not the expected hostname. I think a better name for the variable might be TEST_INSTALL_HOSTNAME
depending on the use boot params are set and different hostnames can be expected here.